_includes/head.html

```diff
@@ -0,0 +1,14 @@
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    {%- seo -%}
+    <link rel="stylesheet" href="{{ "/assets/main.css" | relative_url }}">
+    {%- feed_meta -%}
+    {%- if jekyll.environment == 'production' and site.google_analytics -%}
+      {%- include google-analytics.html -%}
```

Comment:
What is this exactly? Does this mean we'd be using Google analytic?

Reply:
It's copy-pasted from the [source of the template](https://github.com/jekyll/minima/blob/master/_includes/head.html) with the aditional js/css neeed for the dynamic table. I guess this means that if you configure ``site.google_analytics`` then google analytics is included, but for now, this is disabled (or we can remove it entirely from this source file).